packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/fileUploader.tests.js

```diff
@@ -4338,3 +4338,36 @@ QUnit.module('readOnly option', moduleConfig, () => {
 
 });
 
+QUnit.module('dxButton integration', moduleConfig, () => {
+    QUnit.test('dialog should be shown after press enter key on dxButton (T1178836)', function(assert) {
+        if(devices.real().deviceType !== 'desktop') {
+            assert.ok(true, 'keyboard is not supported for not generic devices');
+            return;
+        }
+
+        const $customDialogTrigger = $('<div>').addClass('trigger').appendTo('#qunit-fixture');
```

Comment:
we cannot get rid of this class and init button directly on the `$customDialogTrigger`